pkg/proxy/nftables/proxier_test.go

```diff
@@ -879,14 +892,14 @@ func TestClusterIPGeneral(t *testing.T) {
 			protocol: v1.ProtocolUDP,
 			destIP:   "172.30.0.42",
 			destPort: 80,
-			output:   "",
+			output:   "REJECT",
```

Comment:
YAY but this reminds me there should be packet flow tests for the drop rule too (I think the stuff in https://github.com/kubernetes/kubernetes/commit/4552b2d84c8f0a7c87d29ff683ccb19470468b2b will handle processing that rule; what eventually got merged only handles the "!= { ... }" case, but if you add back the non-!= case it should handle your rule)